_maps/templates/shelter_2.dmm

```diff
@@ -0,0 +1,39 @@
+"a" = (/turf/closed/wall/mineral/titanium/survival/pod,/area/survivalpod)
+"b" = (/obj/structure/sign/mining/survival{tag = "icon-survival (NORTH)"; icon_state = "survival"; dir = 1},/turf/closed/wall/mineral/titanium/survival/pod,/area/survivalpod)
+"c" = (/turf/closed/wall/mineral/titanium/survival,/area/survivalpod)
+"d" = (/obj/structure/sign/mining/survival{dir = 8},/turf/closed/wall/mineral/titanium/survival/pod,/area/survivalpod)
+"e" = (/obj/structure/fans,/turf/open/floor/pod,/area/survivalpod)
+"f" = (/obj/machinery/smartfridge/survival_pod,/turf/open/floor/pod,/area/survivalpod)
+"g" = (/obj/item/device/gps/computer,/turf/open/floor/pod,/area/survivalpod)
+"h" = (/obj/machinery/shower,/obj/structure/curtain,/obj/machinery/door/window/southright,/obj/item/weapon/soap/deluxe,/turf/open/floor/pod,/area/survivalpod)
+"i" = (/obj/structure/window/reinforced/tinted/frosted,/obj/structure/toilet{icon_state = "toilet00"},/obj/machinery/light/small{tag = "icon-bulb1 (EAST)"; icon_state = "bulb1"; dir = 4; pixel_y = 10},/turf/open/floor/pod,/area/survivalpod)
+"j" = (/obj/structure/sign/mining/survival{tag = "icon-survival (EAST)"; icon_state = "survival"; dir = 4},/turf/closed/wall/mineral/titanium/survival/pod,/area/survivalpod)
+"k" = (/obj/machinery/sleeper/survival_pod,/turf/open/floor/pod,/area/survivalpod)
+"l" = (/turf/open/floor/pod,/area/survivalpod)
+"m" = (/obj/structure/window/reinforced{tag = "icon-rwindow (EAST)"; icon_state = "rwindow"; dir = 4},/turf/open/floor/pod,/area/survivalpod)
+"n" = (/turf/open/floor/carpet,/area/survivalpod)
+"o" = (/obj/structure/sink{tag = "icon-sink (EAST)"; icon_state = "sink"; dir = 4; pixel_x = 12},/turf/open/floor/carpet,/area/survivalpod)
+"p" = (/obj/structure/table/survival_pod,/obj/structure/window/reinforced,/obj/item/weapon/storage/firstaid/regular,/obj/item/weapon/storage/firstaid/brute{pixel_x = 5},/turf/open/floor/pod,/area/survivalpod)
```

Comment:
the problem is that it's perfect for a cult to use since they can deploy it anywhere without having to build so it makes for a perfect space cube. It's large enough you could easily put a pylon there for free healing and have room for cultists to move around and convert and other such things. The normal pod is very cramped and only has like 4 or 5 useable spaces to move in, which isn't ideal for cults.